<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MtbUnisIOWindow</class>
 <widget class="QDialog" name="MtbUnisIOWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_inputs"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_outputs">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>331</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Outputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_outputs"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_servos">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>151</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Servos</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_servos"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_servos_manual">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>151</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Manual positioning</string>
   </property>
   <widget class="QComboBox" name="cb_servo_manual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">6</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_manual_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start manual pos.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_end">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>End manual pos.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Servo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Position:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_manual_pos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_minus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_plusplus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">++</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_minusminus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">--</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_save_plus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to pos. +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_save_minus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to pos. -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_manual_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>b_refresh</tabstop>
  <tabstop>cb_servo_manual</tabstop>
  <tabstop>b_manual_start</tabstop>
  <tabstop>b_manual_end</tabstop>
  <tabstop>sb_manual_pos</tabstop>
  <tabstop>b_manual_set</tabstop>
  <tabstop>b_manual_plus</tabstop>
  <tabstop>b_manual_plusplus</tabstop>
  <tabstop>b_manual_minus</tabstop>
  <tabstop>b_manual_minusminus</tabstop>
  <tabstop>b_manual_save_plus</tabstop>
  <tabstop>b_manual_save_minus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
